code/modules/xenobio2/mob/slime/slime procs.dm

```diff
@@ -0,0 +1,104 @@
+/*
+Slime specific procs go here.
+*/
+#define SHINYOVERLAY 0
+#define LIGHTOVERLAY 1	
+#define MAXOVERLAY 2	//Should be 1 + last overlay, to give the chance for matte slimes
+
+/mob/living/simple_animal/xeno/slime/RandomizeTraits()
+	traitdat.traits[TRAIT_XENO_COLDRES] = rand(30,270)
+	traitdat.traits[TRAIT_XENO_HEATRES] = rand(30,270)
+	traitdat.traits[TRAIT_XENO_CHEMVOL] = round(rand(0,40))	//Wow, a slime core with the capacity to hold 2/3rd's a beaker's worth of chemicals.
```

Comment:
I honestly just threw some values in there, I still need to adjust certain things present in the code.